### PR TITLE
Homolage containers tab to Content Type tabs

### LINKED ISSUE #26197: [Containers] Fix Tabs and Rows Variables Height
### User Story
As a dotCMS user, I want to be able to adjust the height of the tabs and rows in the containers portlet, so I can better organize my content.

### Acceptance Criteria

```[tasklist]
### Tasks
- [x] Fix tabs in containers, [see image](https://github.com/dotCMS/core/assets/751424/20d9cf88-0e2c-4d66-a265-8f92604cb452)
- [ ] Field rows should have padding of 0.25rem, [see image](https://github.com/dotCMS/core/assets/751424/2f71cf5e-e685-45c3-9bb6-4ea1539d34e0)
```

### Proposed Objective
Core Feature

### Proposed Priority
Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.:
N/A

### Assumptions & Initiation Needs
N/A

### Quality Assurance Notes & Workarounds
N/A

### Sub-Tasks & Estimates:
N/A

### PR DESCRIPTION
**Pull request by KevinDavilaDotCMS**
### Summary
🎨🛠️📝

This pull request enhances the UI of the container code editor and the add variable form. It improves the styling of the tabs, the code editor, and the form elements using SCSS. It affects the files `dot-container-code.component.scss` and `dot-add-variable.component.scss`.

> _`dot-add-variable`_
> _Aligns its form in the center_
> _A neat spring cleaning_

### Walkthrough
*  Align form elements vertically in the center of the container form ([link](https://github.com/dotCMS/core/pull/26328/files?diff=unified&w=0#diff-d5edd0e40eead03b1f7393bf8ae908de29bd1d7abf8845cebcd57ba18a380296L30-R30))
*  Remove bottom border and add lighter top and bottom border to hovered tabs ([link](https://github.com/dotCMS/core/pull/26328/files?diff=unified&w=0#diff-8487dfe45ff8a0a95621c288f58247aac38edd44aa95f2a638f8e79332eb9a3bL71-R73))
*  Make code editor fill the available height of the container ([link](https://github.com/dotCMS/core/pull/26328/files?diff=unified&w=0#diff-8487dfe45ff8a0a95621c288f58247aac38edd44aa95f2a638f8e79332eb9a3bR88))

https://github.com/dotCMS/core/assets/751424/80f919cd-32f1-40dc-a12b-a103901961b3

Image

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss [hunk 1]
```diff
@@ -68,12 +68,9 @@
...
     }
 
-    .p-tabview .p-tabview-nav li.p-highlight .p-tabview-nav-link {
-        border-bottom: 4px solid $color-palette-secondary-500;
-    }
-
     .p-tabview .p-tabview-nav li:not(.p-highlight):not(.p-disabled):hover .p-tabview-nav-link {
-        border-bottom: 4px solid $color-palette-secondary-500;
+        border-top: 1px solid #afb3c0;
+        border-bottom: 1px solid #afb3c0;
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:73]: We should use variables here and anywhere else we are uding this color.

Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:73]: Also please check the height with the one in content types and anything else that looks different here, this tabs should looks the same as the one in content types.

Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:73]: I think there are custom tab code here that we don't need anymore.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss (hunk 1 above)
Comment (zJaaal) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:73]: check `_colors.scss` I think this is both `$color-palette-gray` and  `$color-palette-gray-500`

**Review by zJaaal — COMMENTED** [review]
See my comment

**Review by rjvelazco — COMMENTED** [review]
Use `CSS` variable for colors

**Review by zJaaal — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**